README.md

```diff
@@ -33,12 +33,16 @@ npm install --save ethdeploy
 
 ## Example
 
-Checkout the ethdeploy [example](/example/index.js) provided. This will deploy a bunch of contracts with an ethdeploy setup and config.
+Checkout the ethdeploy [example](/example/index.js) provided. This will launch a testrpc server (a dummy ethereum api) in the background, and then deploy a bunch of contracts.
 
 ```
 cd example
 npm install
-npm run example
+(cd node_modules/ethdeploy/ && npm install)
+npm install -g eslint
+node_modules/.bin/testrpc & 
+[some initialization of the 'input' address]
```

Comment:
Can you show exactly how you would do this? Or better, add a script that adds the input transaction, and spits out a new environments.json ?